spec/nine_one_one/notifier_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'spec_helper'
+
+describe NineOneOne::Notifier do
+  subject(:notifier) { described_class.new(config) }
+  let(:config) { NineOneOne::Configuration.new }
+
+  describe '.emergency_service' do
+    context 'when send_pagers is true' do
```

Comment:
you could define:
subject { notifier.emergency_service }

and then simplyfy use one-liner:
it { is_expected.to to be_a NineOneOne::PagerDutyService }